tests/tests.rs

```diff
@@ -987,6 +987,57 @@ fn assert_exec_output(exec_style: &str) {
     }
 }
 
+/// Shell script execution using -exec
+#[test]
+fn test_exec_batch() {
+    assert_exec_batch_output("--exec-batch");
+}
+
+// Shell script execution using -x
+#[test]
+fn test_exec_batch_short_arg() {
+    assert_exec_batch_output("-X");
+}
+
+#[cfg(test)]
+fn assert_exec_batch_output(exec_style: &str) {
+    let (te, abs_path) = get_test_env_with_abs_path(DEFAULT_DIRS, DEFAULT_FILES);
+    let te = te.normalize_line(true);
+
+    // TODO Windows tests: D:file.txt \file.txt \\server\share\file.txt ...
```

Comment:
I'm okay with leaving the Windows tests as To Do, but what does this comment describe?

Reply:
I must admit to a copy/paste (incl. this comment).
The `cfg!(windows)` clause is already present in the `exec` test, and being on linux right now, I couldn't try it out.
I guess the comment is mainly linked to the first test (absolute paths), which may have several forms on windows depending of file location (drive, network,…). But this is just a theory.